src_erl/Communication_Layer/http_Nerlserver/src/nerlNetServer_app.erl

```diff
@@ -45,17 +45,17 @@
 
 start(_StartType, _StartArgs) ->
      HostName = getHostName(),
-     %HostName = "host_1",
+     %HostName = "127.0.0.1",
      %HostName = "127.0.0.1",
      io:format("My HostName: ~p~n",[HostName]),
 
     %Create a listener that waits for a message from python about the adresses of the wanted json
-    %createNerlnetInitiator(HostName),
+    createNerlnetInitiator(HostName),
     receive 
 
         {jsonAddress,MSG} -> {ArchitectureAdderess,CommunicationMapAdderess} = MSG
         %%TODO remove this "after" part when python is ready to send jsonPaths
-        after 10000 ->   JsonPath = "../../../jsonPath",
+        after 1000000 ->   JsonPath = "../../../jsonPath",
```

Comment:
Add comment - what is the purpose of this PR
better to use macro (?TIMEOUT_SOMETHING) 

@kapelnik